docs/conf.py

```diff
@@ -10,7 +10,7 @@
 
 
 docs_dir = os.path.abspath(os.path.dirname(__file__))
-base_dir = os.path.join(docs_dir, "..")
+base_dir = os.path.abspath(os.path.join(docs_dir, ".."))
```

Comment:
While you're correcting the `base_dir`, maybe get rid of `docs_dir` altogether since it is only used once below and it needs the `base_dir` anyway :smile: